_posts/2021-01-12-magic-control.adoc

```diff
@@ -0,0 +1,281 @@
+---
+layout: post
+title: Bored with magic tricks?
+date: 2021-01-06 # Update the date before publication
+tags: vertx
+synopsis: How to control the amount of magic from Quarkus
+author: cescoffier
+---
+
+Just before my PTO, someone told me: '_I don't like magic._'
```

Comment:
I thought that would turn into a Carla Bruni song.

Reply:
Can we get the Carla Bruni version too? I’m curious now.

Reply:
"Someone told me... that you didn't like magiiiiic"

https://www.youtube.com/watch?v=EelX_LwPHbA